app/V1Module/security/UserStorage.php

```diff
@@ -35,12 +35,13 @@ public function __construct(AccessManager $accessManager, Nette\Http\IRequest $h
     }
 
     /**
-     * @param bool
-     * @return void
+     * @param bool $state
+     * @return static
```

Comment:
static?

Reply:
Changed to `self` which is more self-explaining